alvr/client_core/resources/staging_fragment.glsl

```diff
@@ -3,9 +3,17 @@
 
 uniform samplerExternalOES tex;
 
+// Convert from limited colors to full
+const float LIMITED_MIN = 0.06274509803; // 16.0 / 255.0
+const float LIMITED_MAX = 0.92156862745; // 235.0 / 255.0
+
 in vec2 uv;
 out vec4 out_color;
 
 void main() {
-    out_color = texture(tex, uv);
+    vec3 color = texture(tex, uv).rgb;
+#ifdef FIX_LIMITED_RANGE
```

Comment:
I feel like you should probably use specialization constants for this (https://webgpufundamentals.org/webgpu/lessons/webgpu-constants.html), it's a lot cleaner than doing text replaces

Reply:
WGPU used this, but this is GLES so it cannot unfortunately.

Reply:
Oh right sorry, I forgot we still have some normal gles in there (I have been writing 3 different shading languages in the past two weeks I have no clue how they look like anymore)